extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js

```diff
@@ -25,7 +25,14 @@ require([
 ) {
 	var allowedPlayerImpressionsPerSession = videoDetails.impressionsPerSession || 1;
 
-	if (!canPlayVideo()) {
+	window.canPlayVideo = function () {
```

Comment:
A nice-to-have would be going through the file and replacing all `window` usages with `win` which is being taken-care by require. If it doesn't make sense in your opinion (for example because the module will only be loaded in browsers) I'd suggest to removing the `win` reference and using global `window` everywhere in the file in sake of consistency.